### PR TITLE
[test_0]:HelloWorld pr for users.

### PR DESCRIPTION
**Pull request by wsh-wood**
### What?
- brief: 第一次尝试提交pr
- author: wsh_wood 

### What changes are included in this PR?

添加了一个test目录，包括src子目录，变更了md文件的信息。

### Are these changes tested?

1.已对项目代码进行了编译运行测试。
image

### other information
目前没有团队，期望加入一个团队。如果没有希望能创建一个"Wooden_house"为名的团队。

### CODE REVIEW

**Review by wsh-wood — COMMENTED** [review]
1

**Comment by Light-City** [comment]
Test-> test

**Comment by Light-City** [comment]
dir
> Test-> test

**Comment by Light-City** [comment]
pr title is wrong

**Comment by wsh-wood** [comment]
> pr title is wrong

已对pr title和文件夹名称做了修改，烦请过审。

**Comment by Light-City** [comment]
wsh-wood ->wsh_wood

use underscore.

**Comment by wsh-wood** [comment]
> wsh-wood ->wsh_wood
> 
> use underscore.

Thank you for reminding me, I have made the correction.

File: stl/test/wsh_wood/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <vector>
+#include <string>
+
+int main() {
+    std::vector<char> data;
```
Comment (martinx) [inline stl/test/wsh_wood/src/main.cpp:6]: Indent 2 spaces

Comment (wsh-wood) [reply stl/test/wsh_wood/src/main.cpp:6]: Thank you for your correction, I overlooked that the default 4 indentation in vscode does not comply with the project specifications.

Comment (wsh-wood) [reply stl/test/wsh_wood/src/main.cpp:6]: > Indent 2 spaces
Teacher, do I still need to review it after I have made the changes?

**Comment by martinx** [comment]
You have not submitted the code with modified indentation, please check your local files, thx.

**Comment by wsh-wood** [comment]
> You have not submitted the code with modified indentation, please check your local files, thx.
Teacher, I'm very sorry. There was an error when I submitted it and I didn't notice. It has been corrected, sorry for the trouble.


> 



> You have not submitted the code with modified indentation, please check your local files, thx.

**Comment by martinx** [comment]
LGTM

**Pull request closed without merging**